build.gradle

```diff
@@ -218,6 +218,7 @@ project(':prebuild') {
 
         description = 'Generate Cacerts from JDK source'
         classpath = files(classPath)
```

Comment:
Can you add a comment on where these settings came from?

Reply:
Updated.